### DISCUSSION TITLE
当复制图片到note时，base64一下，把图片变成文本存在文本文件里

### DISCUSSION THREAD

**Discussion by sluke** [tw93/MiaoYan#311] [Ideas]
如题，这样可以在一定程度上脱离图床

**Comment by tw93** [comment]
其实这里我很推荐使用 https://github.com/gee1k/uPic 这个工具，结合GitHub来使用当做一个图传，更好的可以使用云厂商的oss服务

**Reply by sluke** [reply]
这样会多一个工具，还少一种文件类型，而且文本打包压缩很方便
